po/it.po

```diff
@@ -449,7 +449,7 @@ msgid ""
 "At the end of a translation, simply press <Enter> to continue with the next "
 "one."
 msgstr ""
-"Alla fine della traduzione, premere semplicemente <Invio> per continuare con "
+"Alla fine della traduzione, premere semplicemente <Enter> per continuare con "
```

Comment:
The problem with correcting any of these 'XML' is that they probably need a review of the whole string from a language perspective.  Lets land this one, but be aware that you don't know the software enough to know if someone has just reused another string.